README.md

```diff
@@ -1,3 +1,62 @@
-# sonarqube (Sourcegraph extension)
+# [Sonarqube](https://www.sonarqube.org/) Sourcegraph extension
```

Comment:
I was curious what our style guide says about naming integrations. It actually does not address this particular kind format of a name for an integration extension. Maybe we should add this naming pattern to the handbook ("$X Sourcegraph extension"). https://about.sourcegraph.com/handbook/communication/content_guidelines/style_and_mechanics#writing-about-ourselves

Reply:
Good question. The generator generates the "(Sourcegraph extension)", but I find that ugly. Feel free to do a PR to the handbook and the generator!